docs/source/using/docker-daemon.adoc

```diff
@@ -6,11 +6,6 @@ include::variables.adoc[]
 :toc-title:
 :toclevels: 1
 
-toc::[]
-
-[[reusing-docker-daemon]]
-== Reusing the Docker Daemon
```

Comment:
I imagine both this heading and the table of contents were removed due to this document only having a single heading/topic. This is inconsistent with the rest of the documentation, though, and also introduces more work later if another heading needs to be added to this document in the future.

Personally, I'd prefer to keep `toc::[]` and the heading even if it seems like overkill for a single topic.

Reply:
For me it is really about the L&F. IMO it does not look good w/. Adding a [toc] when required is trivial.

Reply:
Understood, and I agree that it feels somewhat awkward.

In this case, I like the consistency and the fact that adding a new header would require no more effort than it would in any other topic within the documentation. Removing the TOC and header means that we'll have to remember that this one particular part of the documentation is different. There's also the `xref` in **_contributing/writing_docs.adoc_** to keep in mind, which would also have to be updated if we add the TOC and header back in at any point.

I just think that it would make more sense to avoid potential errors in the future. I don't feel particularly strongly about this, though, so if anyone else agrees that we should remove the TOC from this particular topic, I don't mind.